### DISCUSSION TITLE
Could we use m5stack's Unit CamS3 Wi-Fi Camera?

### DISCUSSION THREAD

**Discussion by p0macs** [jomjol/AI-on-the-edge-device#3432] [General]
### The Feature

Hi,
Recently I got this one:
https://shop.m5stack.com/products/unit-cams3-wi-fi-camera-5mp

Is there any chance we could run AI-on-the-edge-device on this hardware?

Thank you for your great work!

**Comment by caco3** [comment]
There are already people who tried to compile it for the m5stack, see https://github.com/jomjol/AI-on-the-edge-device/discussions/982 and other discussions.
But I don't know the outcome.

**Reply by SybexX** [reply]
The “Unit CamS3 Wi-Fi Camera 5MP” is an ESP32-S3 and is not supported.
